src/main/java/uk/gov/companieshouse/api/strikeoffobjections/processor/ObjectionProcessor.java

```diff
@@ -53,9 +55,10 @@ public ObjectionProcessor(IEmailService emailService,
      *
      * @param objection     the objection to process
      * @param httpRequestId http request id used for logging
+     * @param out an ObjectionPatch that is updated as the objection is processed
      * @throws InvalidObjectionStatusException if the Objection is not currently in status SUBMITTED when this is called
      */
-    public void process(Objection objection, String httpRequestId)
+    public void process(Objection objection, String httpRequestId, ObjectionPatch out)
```

Comment:
Could we rename the patch from "out" to something more descriptive, "such as outgongPatch" or "outgoingPatch".